experimental/PropertyDDS/packages/property-inspector-table/package.json

```diff
@@ -114,11 +114,11 @@
 		"webpack-merge": "^5.8.0"
 	},
 	"peerDependencies": {
-		"@fluid-experimental/property-binder": "^1.0.0",
-		"@fluid-experimental/property-changeset": "^1.0.0",
-		"@fluid-experimental/property-dds": "^1.0.0",
-		"@fluid-experimental/property-properties": "^1.0.0",
-		"@fluid-experimental/property-proxy": "^1.0.0",
+		"@fluid-experimental/property-binder": ">=2.0.0-internal.3.1.0 <2.0.0-internal.4.0.0",
+		"@fluid-experimental/property-changeset": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
+		"@fluid-experimental/property-dds": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
+		"@fluid-experimental/property-properties": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
+		"@fluid-experimental/property-proxy": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
```

Comment:
In 9.3.2 it should tell you that it can't fix them (where before it silently ignored them) but it _should_ know when they mismatch and tell you there's a mismatch it cannot fix. Thanks for the repro branch, I'll see what I can find on this and the other problems.